R/data.R

```diff
@@ -55,11 +55,50 @@
 #' @source \url{https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005697#sec020}
 "contact_other"
 
-#' mort_server_default
+#' mort_sever_default
 #' 
-#' Two data points per age group: infection fatality rate (ifr) and 
-#' infection hospitalisation rate (ihr)???
-#' ifr has been scaled so the maximum value is 1?
-#' ihr is a percentage
+#' Default infection fatality rate (ifr) and infection hospitalisation rate 
+#' (ihr) per age group. ifr has been scaled so the maximum value is 1. 
+#' 
+#' @format A dataframe with three variables
+#' \describe{
+#' \item{age_category}{0-4 years, 5-9 years, etc. up to and including 100+ years old}
+#' \item{ifr}{fraction of infections that leads to death}
+#' \item{ihr}{fraction of infections that leads to hospitalisation}
+#' }
+"mort_sever_default"
 
+#' population
+#' 
+#' Per country and per age group the number of individuals in the population, 
+#' the number birth rate and the death rate. 
+#' 
+#' @format A dataframe with five variables
+#' \describe{
+#' \item{country}{country}
+#' \item{age_category}{0-4 years, 5-9 years, etc. up to and including 100+ years old}
+#' \item{pop}{the number of individuals in this country in this age group}
+#' \item{birth}{number of births per person per day}
+#' \item{death}{number of deaths per person per day}
+#' }
+"population"
 
+#' cases
+#' 
+#' NOTE: I think this data is actually imported from the excel file. The sheets
+#' "Cases" or "Epidemiology" provide the same information and are imported in the
+#' covidage_v16.8.R script. The line where data_Como.Rdata is loaded is commented
+#' out by default. However, the excel file only provides data for one
+#' country, not for all like this data set does.
+#' 
+#' Data on the number of cases and deaths per country per day from 31-12-2019.
```

Comment:
is it data for just this date (for initialising purposes) ? or this day onwards (for model fitting purposes)?

If it's this day onwards when does the data run until?